boards/arm/mimxrt685_evk/doc/index.rst

```diff
@@ -129,12 +131,12 @@ functionality of a pin.
 +---------+-----------------+----------------------------+
 | PIO0_21 | I2S             | I2S TX SCK                 |
 +---------+-----------------+----------------------------+
+| PIO0_9  | I2S             | I2S DATAIN                 |
++---------+-----------------+----------------------------+
 | PIO0_29 | USART           | USART TX                   |
 +---------+-----------------+----------------------------+
 | PIO0_30 | USART           | USART RX                   |
 +---------+-----------------+----------------------------+
-| PIO0_9  | I2S             | I2S DATAIN                 |
-+---------+-----------------+----------------------------+
```

Comment:
This change is not related to PWM

Reply:
For some reason the I2S related items were not grouped together. I made this change while adding the PWM information.

Reply:
It's ok to move the I2S items together, just do it in a separate commit

Reply:
Done, PR https://github.com/zephyrproject-rtos/zephyr/pull/36809 created